src/blog/2024/08/comparing-dashboard-2-with-uibuilder.md

```diff
@@ -0,0 +1,233 @@
+
+## How much dynamic insertion or update it support?
+
+### Node-RED Dashboard
+
+Node-RED Dashboard 2.0 supports dynamically updating the UI via the msg object. Each UI widget supports the `msg.ui_update` property with a JSON object that allows you to update the UI components dynamically. For example, you can update form fields based on user input, dynamically insert or update options in a dropdown, or change the appearance of the UI by sending css classes.
+
+### Ui-builder
+
+UI-Builder also supports dynamic UI updates. Similar to Node-RED Dashboard, you can use messages to control the state and content of UI elements.
+
+## Is creating visualizations easy in the Dashboard? ?
+
+### Node-RED Dashboard 2.0
+
+Visualization is a core use case for Node-RED, and Node-RED Dashboard 2.0 makes it easy with its built-in [chart](https://dashboard.flowfuse.com/nodes/widgets/ui-chart.html) and [gauge widgets](https://dashboard.flowfuse.com/nodes/widgets/ui-gauge.html). The chart widgets support various types, including line, bar, scatter, pie, and donut charts, while the gauge widgets offer options like tile, 3/4 gauge, and half gauge. This range of built-in options simplifies the creation of visualizations. Additionally, you can use Vuetify or other third-party libraries in the ui-template node if you need different types of charts.
+
+### UI-Builder
+
+UI-Builder does not provide built-in charting options. However, you can add charts using third-party libraries, which can make the visualization setup more complex and time-consuming. This approach requires additional effort to integrate and configure the libraries, potentially increasing the complexity of creating visualizations.
+
+## How Fast and Easy is It to Communicate with Node-RED Instances?
+
+### Node-RED Dashboard 2.0
+
+Node-RED Dashboard 2.0 uses WebSockets for communication with Node-RED, enabling real-time and faster interactions. Most widgets uses the standard `msg.payload` for communication that makes it super easy to send the data from Node-RED to client and vice-versa.
+
+### Ui-builder 
+
+The ui-builder also employs WebSockets for communication, providing real-time and fast interactions. However, it utilizes different msg properties, which can be complex for users and might increase the number of nodes unnecessarily for changing the property names.
+
+## How many Web Layout Does It Support?
```

Comment:
This is a mis-leading header as UI Builder offers complete customisation - we do not.

"Building Layouts" would be more appropriate